src/node_report.cc

```diff
@@ -871,6 +874,13 @@ std::string TriggerNodeReport(Isolate* isolate,
       filename = *DiagnosticFilename(
           env != nullptr ? env->thread_id() : 0, "report", "json");
     }
+    if (env != nullptr) {
```

Comment:
I'm not entirely sure about when the env is null. Would that happen only when OOM or another crash reporter happens? If so, what would be the best way to throw an exception here?

Maybe @gireeshpunathil has some thoughts on this?